analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/TestHelper.cs

```diff
@@ -49,37 +49,43 @@ internal static class TestHelper
     <OutPath>{2}</OutPath>
 </SonarProjectConfig>";
 
-        public static (SyntaxTree, SemanticModel) CompileIgnoreErrors(string snippet, bool isCSharp = true, params MetadataReference[] additionalReferences) =>
-            Compile(snippet, true, isCSharp, additionalReferences);
+        public static (SyntaxTree Tree, SemanticModel Model) CompileIgnoreErrorsCS(string snippet, params MetadataReference[] additionalReferences) =>
+            Compile(snippet, true, AnalyzerLanguage.CSharp, additionalReferences);
 
-        public static (SyntaxTree, SemanticModel) Compile(string snippet, bool isCSharp = true, params MetadataReference[] additionalReferences) =>
-            Compile(snippet, false, isCSharp, additionalReferences);
+        public static (SyntaxTree Tree, SemanticModel Model) CompileCS(string snippet, params MetadataReference[] additionalReferences) =>
+            Compile(snippet, false, AnalyzerLanguage.CSharp, additionalReferences);
 
-        public static (SyntaxTree, SemanticModel) Compile(string snippet, bool ignoreErrors, bool isCSharp, params MetadataReference[] additionalReferences)
+        public static (SyntaxTree Tree, SemanticModel Model) CompileVB(string snippet, params MetadataReference[] additionalReferences) =>
+            Compile(snippet, false, AnalyzerLanguage.VisualBasic, additionalReferences);
+
+        public static (SyntaxTree Tree, SemanticModel Model) Compile(string snippet, bool ignoreErrors, AnalyzerLanguage language, params MetadataReference[] additionalReferences)
```

Comment:
`Compile` and `CompileCfg` methods should be private as there are other ` AnalyzerLanguages` that are not handled in the method. Where those methods are used now the CS and VB variants could be used to replace it.

Reply:
I've replaced the single usage that could be simplified.